### ISSUE TITLE
Syntax change in response from Tahoma

### ISSUE THREAD

**Issue by qmiloo** [Somfy-Developer/Somfy-TaHoma-Developer-Mode#142]
I'm sorry if there was already a topic around this and I didn't find it.
I have a problem reading the statuses of the blinds. From time to time, Somfy changes the lines containing information about the position of the blinds/slats, which makes it impossible for me to obtain information about the status without providing chnages to script. Has anyone encountered this problem before? Any chance to fix this?

**Comment by BernoldNL** [comment]
Can you indicate what changed? Is it a specific status and if so, which one? Otherwise, could you give an example?

Only thing I noticed is that the order of the elements within the JSON change from time to time. But I would not call that a syntax change.

**Comment by qmiloo** [comment]
Yes, sorry you're right. It's order of elements presenting position of blinds and slates. Any chance that this can be fix somehow?

**Comment by vhenriet-sfy** [comment]
Hi @qmiloo,

Just a quick note to mention that JSON keys don't have a defined order. To ensure proper interpretation, it's recommended to use a standard JSON parser, as it will handle the structure reliably without relying on key order.

If you can share more details about how you're working with the API, we can offer more specific help.

**Comment by qmiloo** [comment]
Hi,

will try as best i can.
I have created HttpRequest to pickup whole setup for my blinds

![C80312D5-EBF7-493E-B643-F68F691E1681_1_201_a](https://github.com/user-attachments/assets/a4b63521-43f0-4273-b3ce-e226db2f0369)

On HttpRespons from that request i have created script which is running accordingly. I'm reading via it blinds(green) and slate(blue) position.

![91A8FC44-7FC7-4AFF-8925-28C5FBD3F4C8_4_5005_c](https://github.com/user-attachments/assets/7da1d02f-2030-44c3-8f25-660e83b757d2)

and this is saved under the user feture blinds(green) and slate(blue) position and blinds io number (orange)
![30531CE8-0F31-4FE5-BD11-BDCCB20F3FA3_4_5005_c](https://github.com/user-attachments/assets/e6d9d948-bd09-4527-a00c-9fe68fbdc9a7)

As you can see on the above screenshot something has change for slate position in the order because now i'm taking some table name - not this what i want and what i was able to take previous.
Now i have to change states(yellow) in script to read correct information...of course only when i will find proper line checking previously via Postman. In most of the cases it will be +1  or -1 from this what was previous in the script and  for sure currently what i can see under blind position is the information from slate position. It means that every time i need to correct both states[]

![91A8FC44-7FC7-4AFF-8925-28C5FBD3F4C8_4_5005_c](https://github.com/user-attachments/assets/68a2c738-57b7-4185-bc7d-5b9166decd26)

Please let me know if something is unclear 

Thanks in advance
